blocks/rich-text/index.js

```diff
@@ -558,6 +558,38 @@ export class RichText extends Component {
 		if ( keyCode === BACKSPACE ) {
 			this.onChange();
 		}
+
+		// `scrollToCaret` is called on `nodechange`, whereas calling it on
+		// `keyup` *when* moving to a new RichText element results in incorrect
```

Comment:
Why do we assume backspace or enter would result in a new RichText element? Doesn't this depend on presence of `onSplit` and `onMerge` props?

Wondering also if this is a behavior that ought to be handled by individual instances of RichText, vs. a global handler (WritingFlow?)

Reply:
> Doesn't this depend on presence of onSplit and onMerge props?

It does; I've been trying out different heuristics but wanted to maintain momentum. Not sure what to converge on. In practice, I'm seeing good behaviors on paragraph, quote and list, which cover a decent set of `multiline / onSplit / onMerge` combinations.

> WritingFlow?

On one hand, it makes architectural sense; on the other, RichText may be the best entity to track the caret in its own `editor`, so I'm not sure. It'd be great to have this in the next release, though, which is why I followed the approach of the parent PR.